packages/babel-parser/test/fixtures/typescript/disallow-jsx-ambiguity/type-parameter-babel-7/input.ts

```diff
@@ -1,2 +1,3 @@
 <T>() => 1;
 <T>(x) => 1;
+<T = unknown>(x) => 1;
```

Comment:
Why is this ambiguous?

Reply:
Heh, had the same doubt - i dont know. It’s treated as ambigous by TS though 🤷‍♂️

Reply:
This sounds like a bug, but yeah we should match their behavior.